CMakeLists.txt

```diff
@@ -432,13 +432,16 @@ OCV_OPTION(WITH_IMGCODEC_PXM "Include PNM (PBM,PGM,PPM) and PAM formats support"
 OCV_OPTION(WITH_QUIRC "Include library QR-code decoding" ON
   VISIBLE_IF TRUE
   VERIFY HAVE_QUIRC)
+OCV_OPTION(WITH_TENGINE "Include Arm Inference Tengine support" ON
+  VISIBLE_IF (ARM OR AARCH64) AND UNIX AND NOT ANDROID AND NOT IOS
```

Comment:
> ON

`NOT ANDROID`

Reply:
@alalek, can you please decrypt this comment? :)

Reply:
`VISIBLE_IF` is for "complete" filtering of platforms. This filter "hides" the option.

Android can be use used, but should be configured for proper version.
So right place for "NOT ANDROID" "by default" is ON/OFF condition.